packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -641,6 +641,20 @@ export function useMemo<T>(
   return nextValue;
 }
 
+// Single source of truth that keeps various hooks APIs in-sync.
+export type DispatcherHooks = {
```

Comment:
Wasn't sure what to call this thing, since it's just the hooks portion of the dispatcher. Any better suggestions?